.github/workflows/java8_integration_tests.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - name: "Upload Code Coverage Report"
         uses: codecov/codecov-action@v1
         # skip generating+uploading coverage on these matrix modules because they are flaky
-        if: ${{ matrix.modules != 'alluxio.server.ft.**,!alluxio.server.ft.journal.raft.**,!alluxio.server.ft.journal.ufs.**' && matrix.modules != 'alluxio.client.cli.**,!alluxio.client.cli.fs.**' }}
+        if: ${{ always() && matrix.modules != 'alluxio.server.ft.**,!alluxio.server.ft.journal.raft.**,!alluxio.server.ft.journal.ufs.**' && matrix.modules != 'alluxio.client.cli.**,!alluxio.client.cli.fs.**' }}
```

Comment:
isn't this essentially the same as saying `(true && condition)` which logically simplifies to `(condition)`? ditto for other line

Reply:
Neh.. This is actually not the same.. Take a look at https://github.community/t/multiple-if-conditional-with-always-is-always-true/128840. The way they handle this is a little odd.

Reply:
weird...